README.md

```diff
@@ -27,12 +41,39 @@ await Name.publish(revision, name.key)
 
 See https://github.com/web3-storage/w3name/blob/main/packages/client/README.md for a guide to using the js client for the first time.
 
+Full API reference doc for the JS client are available at https://web3-storage.github.io/w3name
+
+You can also find a [how-to guide for working with the JS client](https://web3.storage/docs/how-tos/w3name/) included in the [Web3.Storage](https://web3.storage) documentation.
+
 ### cURL
 
-TODO: Add cURL instructions here!
+You can easily get the current value for any name by sending an HTTP request to `https://name.web3.storage/name/:key`, where `:key` is the name string. This will return a JSON object with the following shape:
+
+```json
+{
+  "value": "the current value, as a string",
+  "record": "the full IPNS record for the current value, encoded to a binary form and base64pad encoded"
+}
+```
+
+Try pasting this into a terminal with `curl` installed:
+
+```shell
+curl https://name.web3.storage/name/k51qzi5uqu5dlcuzv5xhg1zqn48gobcvn2mx13uoig7zfj8rz6zvqdxsugka9z
+```
+
+You should see output similar to this:
+
+```json
+{
+  "value": "/ipfs/bafkreigbpn5osrexvl56ogyp5kivof2tmknkssk5odebqqeu23a22bcntu",
+  "record": "long base64 string, omitted for brevity..."
+}
+```
 
-**See https://github.com/web3-storage/w3name/tree/main/packages/api for our complete documentation 📖🔍**
+**See https://github.com/web3-storage/w3name/tree/main/packages/api for documentation on other available API endpoints 📖🔍**
 
+<!-- TODO: add link to swagger api docs once published -->
```

Comment:
Quick note: am working on this currently, got a ticket over at https://github.com/web3-storage/web3.storage/issues/1722 
Will link this to the http swagger docs once done!